docs/reference/esql/functions/avg.asciidoc

```diff
@@ -20,7 +20,7 @@ The average of a numeric expression.
 
 The result is always a `double` no matter the input type.
```

Comment:
> We could mention that individual null values are skipped when computing the avg. (This applies to all our aggs, so that could be also a disclaimer at the top or so.)

General disclaimer could go [here](https://www.elastic.co/guide/en/elasticsearch/reference/master/esql-functions-operators.html#esql-agg-functions)?